server/app.ts

```diff
@@ -0,0 +1,124 @@
+import express from 'express'
```

Comment:
The name/location of this file matches template now, but the contents of this file need to be tackled next to bring in line